### ISSUE TITLE
Destructor is not called for AliasSeq (type tuple)

### ISSUE THREAD

**Issue by JohanEngelen** [ldc-developers/ldc#4427]
Testcase:
```d
import core.stdc.stdio: printf;
alias AliasSeq(TList...) = TList;

int i = 0;
struct A {
    ~this() {
        i++;
    }
}

void main() {
    {
	    AliasSeq!(A, A) params;
    }
    printf("%d\n", i); // prints 0 !
    assert(i == 2); // fails!
}
```
https://godbolt.org/z/TGMWczh3E

-vcg-ast output shows the problem that the frontend does not add destructors calls like it usually does:
```d
void main()
{
	{
		(A, A) params;
               //  there should be two explicit destructors calls here (~this())
	}
	printf("%d\n", i);
	assert(i == 2);
	return 0;
}
```

**Comment by kinke** [comment]
This was fixed in the frontend, right?

**Comment by JohanEngelen** [comment]
Yep, fixed in LDC 1.35.0
